tests_e2e/scenarios/runbooks/daily.yml

```diff
@@ -1,29 +1,23 @@
-name: azure
+name: Daily
```

Comment:
I was using the daily.yml both for creating new VMs or use existing ones. With the addition of the combinator to execute multiple distros that is no longer possible.

I split the existing VM setup into a new runbook in the samples directory.